File: Server/aoRssFeed2/aoRssFeed2/Models/Domain/FeedModel.vb

Option Explicit On
Option Strict On

Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Contensive.BaseClasses

Namespace Models.Domain
    Public Class FeedModel
        Public Id As Integer
        Public Name As String
        Public Description As String
        Public Link As String
        Public LogoFilename As String
        Public RSSFilename As String
        Public entryList As List(Of FeedEntryModel)
        'Public EntryCnt As Integer
        'Public EntrySize As Integer
        'Public EntryDateIndex As New Dictionary(Of String, Integer)
    End Class

    Public Class FeedEntryModel
        Public Title As String
        Public Description As String
        Public DatePublish As Date
        Public DateExpires As Date
        Public Link As String
        Public storyContentID As Integer
        'Public storyTableID As Integer
        Public storyRecordID As Integer
        Public PodcastMediaLink As String
    End Class
End Namespace

File: Server/aoRss/Models/Domain/FeedModel.cs




using System;
using System.Collections.Generic;

namespace Contensive.Addons.Rss.Models.Domain {
    public class FeedModel {
        public int Id;
        public string Name;
        public string Description;
        public string Link;
        public string LogoFilename;
        public string RSSFilename;
        public List<FeedEntryModel> entryList;
        // Public EntryCnt As Integer
        // Public EntrySize As Integer
        // Public EntryDateIndex As New Dictionary(Of String, Integer)
    }

    public class FeedEntryModel {
        public string Title;
        public string Description;
        public DateTime DatePublish;
        public DateTime DateExpires;
        public string Link;
        public int storyContentID;
        // Public storyTableID As Integer
        public int storyRecordID;
        public string PodcastMediaLink;
    }
}